PuzzleServer.sln

```diff
@@ -10,12 +10,17 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ServerCore.Test", "ServerCo
 		{67B91CCE-4E3D-4F0F-8A76-EAFD37C5F949} = {67B91CCE-4E3D-4F0F-8A76-EAFD37C5F949}
 	EndProjectSection
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Puzzleday", "Puzzleday\Puzzleday.csproj", "{EE66B8FE-FF6B-4D82-B682-E04BADC51511}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Puzzleday", "Puzzleday\Puzzleday.csproj", "{EE66B8FE-FF6B-4D82-B682-E04BADC51511}"
```

Comment:
Were this and the line below supposed to change?

Reply:
I have no idea. I tried to change them back, but they seem determined to be the new guid.